index.md

```diff
@@ -48,6 +48,361 @@ def is_even(num):
+```
+
+Corrected version
+
+```python
+name = "Amy"
+age = "17"
+country = "England"
+
+s = "{} who is {} lives in {}".format(name, age, country) # Error on this line
+```
+
+### E1306: too-few-format-args {#E1306}
+
+For your work in python, we expect you to use one argument for a {}
+for format string. This error is similar to E1120, also opposite to E1305.
+In the following example, we give fewer values than {}. Remember to give a value
+to a {} when we are using format string.
+
+```python
+s = "{} and {}".format("first")  # Error on this line
+```
+
+Corrected version
+
+```python
+s = "{} and {}".format("first", "second")  # Error on this line
+```
+
+### E1310: bad-str-strip-call {#E1310}
+
+For your work in python, we do not expect to see a constant string that
+contains duplicate characters.
+
+```python
+foo = "tests"
+foo.lstrip("java")  # Error on this line
+```
+
+### W1303: missing-format-argument-key {#W1303}
+
+For your work in python, we expect to see format string that uses named fields
+receive required keywords. This error is similar to E1120 and E1306. In the
+following example, we should assign three values for bond, james and act.
+
+```python
+action = '{bond}, {james} {act}'.format(bond='bond', james='james') # Error on this line
+```
+
+### W1305: format-combined-specification {#W1305}
+
+For your work in python, we do not expect to see format string contains both
+automatic field numbering (e.g. ‘{}’) and manual field specification (e.g. ‘{0}’).
+For example, we should not use {} and {index} at the same time. This may be
+confusing for python to recognize which value is for which field.
+
+```python
+s = "{} and {0}".format("a", "b")  # Error on this line
+```
+
+Corrected version
+
+```python
+s = "{} and {}".format("a", "b")  # Error on this line
+```
+or
+
+```python
+s = "{0} and {1}".format("a", "b")  # Error on this line
+```
+
+### W1306: missing-format-attribute {#W1306}
+
+For your work in python, we expect to see the format string uses an attribute
+specifier ({0.length}), and the argument passed for formatting has that attribute.
+In the following example, we know string does not have the length attribute.
+This means that we should never use{string.length} in format string.
+```python
+s = '{.5}'.format('aaabbbccc')  # Error on this line
+```
+
+### W1307: invalid-format-index {#W1307}
+
+For your work in python, we expect to see a format string uses a lookup specifier
+ ({a[1]}) with the argument passed for formatting contain that key as an attribute.
+In the following example, there is no key as 'longitude' in geopoint which means
+that there is no value for {0[longitude]} in the format string.
+
+```python
+geopoint = {'latitude':41.123}
+point = '{0[latitude]} {0[longitude]}'.format(geopoint) # Error on this line
+```
+
+### W1401: anomalous-backslash-in-string {#W1401}
+
+In python, we do not expect to see a backslash is in a literal string but not
```

Comment:
I really doubt this is what the error means. A backslash itself can be escaped, to cause a backslash character to appear in the string (e.g., the string `"\\"` should just be a single character, the backslash). Perhaps look up what the valid Python escape characters are, and then put in a backslash plus a non-valid character.